.github/workflows/publish-studio.yaml

```diff
@@ -0,0 +1,53 @@
+
+name: Publish studio and ingress for release
+
+on:
+  push:
+    tags:
+      - 'v*'
+  release:
+    types: [published]
+
+jobs:
+ 
+  build-containers:
+    if: github.repository == 'scilifelabdatacentre/stackn'
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - dockerfile: ./Dockerfile
```

Comment:
Same comment here about the prefix "serve-" as above.

Reply:
Ah thats a really good point! I'll fix that now!